usr.sbin/sysinst/msg.pm.pt

```diff
@@ -0,0 +1,167 @@
+message raid_nomultidim {Matrizes multidimensionais NAO sao suportadas!}
+message raid_numrow_ask {numRow?}
+message raid_numcol_ask {numCol?}
+message raid_numspare_ask {numSpare?}
+message raid_sectpersu_ask {sectPerSU?}
+message raid_superpar_ask {SUsPerParityUnit?}
+message raid_superrec_ask {SUsPerReconUnit?}
+message raid_disks {Discos em RAID:}
+message vnd_err_menufmt {CAMINHO NAO DEFINIDO!}
+message vnd_assign {ATRIBUIR}
+message vnd_path_fmt {Caminho de arquivo}
+message vnd_assign_fmt {Criar nova imagem}
+message vnd_size_fmt {Tamanho}
+message vnd_ro_fmt {Somente leitura}
+message vnd_geom_fmt {Defini geometria manualmente}
+message vnd_bps_fmt {Bytes por setor}
+message vnd_spt_fmt {Setores por trilha}
+message vnd_tpc_fmt {Trilhas por cilindros}
+message vnd_cyl_fmt {Cilindros}
+message vnd_path_ask {Caminho de arquivo?}
+message vnd_size_ask {Tamanho (MB)?}
+message vnd_bps_ask {Bytes por setor?}
+message vnd_spt_ask {etores por trilha?}
+message vnd_tpc_ask {Trilhas por cilindros?}
+message vnd_cyl_ask {Cilindros}
+message cgd_err_menufmt {DISCO NAO DEFINIDO!}
+message cgd_dev_fmt {Dispositivo de base}
+message cgd_enc_fmt {Criptografia}
+message cgd_key_fmt {Tamanho da chage}
+message cgd_iv_fmt {Algoritmo IV}
+message cgd_keygen_fmt {Gera�ao de chaves}
+message cgd_verif_fmt {M�todo de verifica�ao}
+message lvm_disks {Discos em VG}
+message lvm_err_menufmt {VG VAZIO!}
+message lvm_disks_fmt {PV's}
+message lvm_name_fmt {Nome}
+message lvm_maxlv_fmt {MaxLogicalVolumes}
+message lvm_maxpv_fmt {MaxPhysicalVolumes}
+message lvm_extsiz_fmt {PhysicalExtentSize}
+message lvm_name_ask {Nome?}
+message lvm_maxlv_ask {MaxLogicalVolumes?}
+message lvm_maxpv_ask {MaxPhysicalVolumes?}
+message lvm_extsiz_ask {PhysicalExtentSize (MB)?}
+message lvmlv_menufmt {Volume l�gico}
+message lvmlv_name_fmt {Nome}
+message lvmlv_size_fmt {Tamanho}
+message lvmlv_ro_fmt {Somente leitura}
+message lvmlv_cont_fmt {Cont�gua}
+message lvmlv_extnum_fmt {LogicalExtentsNumber}
+message lvmlv_minor_fmt {Menor n�mero}
+message lvmlv_mirrors_fmt {Espelhamentos}
+message lvmlv_regsiz_fmt {MirrorLogRegionSize}
+message lvmlv_pers_fmt {N�mero menor persistente}
+message lvmlv_readahsect_fmt {ReadAheadSectors}
+message lvmlv_stripes_fmt {Stripes}
+message lvmlv_stripesiz_fmt {Stripesize}
+message lvmlv_zero_fmt {Zerando o primeiro KB}
+message lvmlv_name_ask {Nome?}
+message lvmlv_size_ask {Tamanho (MB)?}
+message lvmlv_extnum_ask {LogicalExtentsNumber?}
+message lvmlv_minor_ask {N�mero menor?}
+message lvmlv_mirrors_ask {Espelhamento?}
+message lvmlv_regsiz_ask {MirrorLogRegionSize?}
+message lvmlv_readahsect_ask {ReadAheadSectors?}
+message lvmlv_stripes_ask {Stripes?}
+
+message notsupported {Opera�ao nao suportada!}
+message edit_parts {Editar parti�oes}
+message switch_parts {Switch partitioning scheme}
+message fmtasraid {Formatar como RAID}
+message fmtaslvm {Formatar como LVM PV}
+message encrypt {Criptografar (CGD)}
+message erase {Apaga com seguran�a}
```

Comment:
question: aren't you inferring too much meaning from this?

Reply:
Inferring!? What do you mean?

Reply:
Going from simply "Erase" to "Safely erase"?

Reply:
The English version is: Safe erase. (https://github.com/danielsoro/netbsd-src/blob/trunk/usr.sbin/sysinst/msg.pm.en#L162)